### PR TITLE
Make TenantAuthorization easier to extend

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: flask_resty_tenants/authorization.py [hunk 1]
```diff
@@ -12,65 +13,65 @@
 # -----------------------------------------------------------------------------
 
 
-class TenantCredentials(dict):
-
-    def __init__(self, metadata_location='app_metadata'):
```
Comment (itajaja) [inline flask_resty_tenants/authorization.py:17]: I don't even know what that `metadata_location` argument was doing there......

**Review by itajaja — COMMENTED** [review]
am curious: what's the use case for the refactoring?

File: flask_resty_tenants/authorization.py (hunk 1 above)
Comment (taion) [reply flask_resty_tenants/authorization.py:17]: It wasn't doing anything

Comment (itajaja) [reply flask_resty_tenants/authorization.py:17]: yeah i know :P

**Pull request merged**